docs/src/create_reference_md.py

```diff
@@ -142,6 +145,8 @@ def source_section(source: Source, source_key: str, level: int) -> str:
     """Return the source as Markdown section."""
     markdown = markdown_header(source.name, level, index=True)
     markdown += markdown_paragraph(source.description)
+    if documentation := (source.documentation and source.documentation.get("generic")):
```

Comment:
I think `source.documentation.get("generic") if source.documentation` would be more clear

Reply:
That's not valid Python? I've changed it like this so the walrus assignment doesn't depend on the fact that `a and b` returns `b` if `a` and `b` are both true-ish.

```suggestion
   if source.documentation and (documentation := source.documentation.get("generic")):
```